credoai/artifacts.py

```diff
@@ -110,36 +119,43 @@ def get_assessment_plan(self):
                 else:
                     passed_metrics.append(m)
             if risk_issue in RISK_ISSUE_MAPPING:
-                update_dictionary(assessment_plan[RISK_ISSUE_MAPPING[risk_issue]],
-                                  {'metrics': passed_metrics})
+                update_dictionary(
+                    assessment_plan[RISK_ISSUE_MAPPING[risk_issue]],
+                    {"metrics": passed_metrics},
+                )
         # alert about missing metrics
         for m in missing_metrics:
-            logging.warning(f"Metric ({m}) is defined in the assessment plan but is not defined by Credo AI.\n"
-                            "Ensure you create a custom Metric (credoai.metrics.Metric) and add it to the\n"
-                            "assessment plan passed to lens")
+            logging.warning(
+                f"Metric ({m}) is defined in the assessment plan but is not defined by Credo AI.\n"
+                "Ensure you create a custom Metric (credoai.metrics.Metric) and add it to the\n"
+                "assessment plan passed to lens"
+            )
         # remove repeated metrics
         for plan in assessment_plan.values():
-            plan['metrics'] = list(set(plan['metrics']))
+            plan["metrics"] = list(set(plan["metrics"]))
         return assessment_plan
 
     def get_policy_checklist(self):
-        return self.assessment_spec.get('policy_questions')
+        return self.assessment_spec.get("policy_questions")
 
     def get_info(self):
         """Return Credo AI Governance IDs"""
         to_return = self.__dict__.copy()
-        del to_return['assessment_spec']
-        del to_return['warning_level']
+        del to_return["assessment_spec"]
+        del to_return["warning_level"]
         return to_return
 
     def get_defined_ids(self):
         """Return IDS that have been defined"""
         return [k for k, v in self.get_info().items() if v]
 
-    def register(self,
-                 model_name=None,
-                 dataset_name=None,
-                 training_dataset_name=None):
+    def register(
```

Comment:
My understanding is that it applies/assigned a template to a model, and not register the template. If it is right, it is rather confusing that `assessment_template` is here.

Reply:
You are right, but I didn't want to require another function call. Gonna leave it like this for now.